opencog/reasoning/pln/rules/deduction.scm

```diff
@@ -8,19 +8,9 @@
 (define pln-rule-deduction
     (BindLink
         (VariableList
-            (TypedVariableLink
                 (VariableNode "$A")
-                (TypeNode "ConceptNode")
-            )
-            (TypedVariableLink
                 (VariableNode "$B")
-                (TypeNode "ConceptNode")
-            )
-            (TypedVariableLink
-                (VariableNode "$C")
-                (TypeNode "ConceptNode")
```

Comment:
it would be best to SatisyingSetLink to the TypedVaraibleLink. Use ListLink to specify multiple atom types for the VariableNode